### ISSUE TITLE
Clump empty slots not working.

### ISSUE THREAD

**Issue by strickland** [arkayenro/arkinventory#1312]
I don't know if I misunderstand this feature, but I'm thinking it shall put all empty slots into one box and show them as a count, just like adibags style? if I am correct then its not working, and if I check this option actually nothing happens.

I'd like to show all empty slots with a number of empty space, if that make any sense.

**Comment by arkayenro** [comment]
no, clumping merges two categories, eg empty slots > cooking with skills > cooking, that way your empty cooking slots are with your cooking items.

if you want to limit the number of visible empty slots you want the set option just before that, stack limit

**Comment by Shojiro0430** [comment]
How would you show the number of empty slots left when you set a stack limit of 1 then? It reduces down to one slot, but then I dont see any way to actually see how many slots you have remaining.

**Comment by arkayenro** [comment]
you cant.  empty slots dont show a count, even if you limit the number displayed.

you have to use the empty slot text that is in the status area

**Comment by dianegall** [comment]
I just wanted to say that this is a life changing tip. Thanks for this.

**Comment by t-mart** [comment]
Thanks for the explanation @arkayenro. For others who want the visual, you do it with this setting:

![image](https://user-images.githubusercontent.com/200012/204471130-6c4dc935-4fc7-4a27-9d7a-712314475042.png)

Note that when I first moved this slider, nothing happened. But **closing and then reopening the bag frame made the slider apply correctly**.
